idl/info.go

```diff
@@ -35,6 +35,5 @@ func (i *Info) Pos(n ast.Node) ast.Position {
 	if pos, ok := ast.Pos(n); ok {
 		return pos
 	}
-	pos := i.nodePositions[n]
-	return ast.Position{Line: pos.Line}
+	return i.nodePositions[n]
```

Comment:
This change should have been part of d7c4d71abe562dc7852dac633a84e1c8130f0985 (#505).